example-app-python-django/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+services:
+
+  roach1:
+    container_name: roach1
+    hostname: roach1
+    image: cockroachdb/cockroach:v22.2.2
+    networks:
+      - roachnet
+    ports:
+      - "26257:26257"
+      - "8080:8080"
+    command: start --insecure --join=roach1,roach2,roach3
+    volumes:
+      - ~/cockroach-data/roach1:/cockroach/cockroach-data
+  
+  roach2:
+    container_name: roach2
+    hostname: roach2
+    image: cockroachdb/cockroach:v22.2.2
+    networks:
+      - roachnet
+    command: start --insecure --join=roach1,roach2,roach3
+    volumes:
+      - ~/cockroach-data/roach2:/cockroach/cockroach-data
+    depends_on:
+      - roach1
+  
+  roach3:
+    container_name: roach3
+    hostname: roach3
+    networks:
+      - roachnet
+    image: cockroachdb/cockroach:v22.2.2
+    command: start --insecure --join=roach1,roach2,roach3
+    volumes:
+      - ~/cockroach-data/roach3:/cockroach/cockroach-data
+    depends_on:
+      - roach1
+
+  weba:
+    build: .
+    extra_hosts:
+      - 'host.docker.internal:host-gateway'
+    ports:
+      - '8000:8080'
+    depends_on:
+      - roach1
+      - roach2
+      - roach3
+

```

Comment:
Add another duplicate container webB, set port mapping to `8001:8080`. Set DB port in environment variables to `26258` so that it connects to roach3 and not roach1.